src/lib_c/arm-linux-gnueabihf/c/unistd.cr

```diff
@@ -8,6 +8,8 @@ lib LibC
   X_OK                =  1
   SC_CLK_TCK          =  2
   SC_NPROCESSORS_ONLN = 58
+  # FIXME: SC_GETGR_R_SIZE_MAX = 69???
+  # FIXME: SC_GETPW_R_SIZE_MAX = 70???
```

Comment:
Are these still relevant?

Reply:
Those should be fixed. They are need for that platform I haven't been able to verify them. Forgot about the need to. Does anyone know or use that platform and can extract the correct values from the includes?

Reply:
With that I will set them as such. I just haven't managed to personally witness it with my own eyes.